internal/app/models/timelineModel/timelines.go

```diff
@@ -0,0 +1,22 @@
+package timelineModel
```

Comment:
In `components/timeline.go`, is `GetTimelineById` going to contain db calls? If so, can we move it here for now (even though its not implemented yet, but to save the trouble)?

Reply:
yeah good point